docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1497,15 +1497,23 @@ The following file types and formats are supported for the Hive connector:
 * RCText (RCFile using ``ColumnarSerDe``)
 * RCBinary (RCFile using ``LazyBinaryColumnarSerDe``)
 * SequenceFile
-* JSON (using ``org.apache.hive.hcatalog.data.JsonSerDe``)
-* CSV (using ``org.apache.hadoop.hive.serde2.OpenCSVSerde``)
+* JSON
+* CSV
 * TextFile
+* Regex
 
-ORC format configuration properties
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Trino has its own native readers and writers for each of these file formats.
```

Comment:
@dain Can you confirm if this is true? Does Trino have native readers and writers for RCText and RCBinary?